### PR TITLE
CHANGELOG for 1.5.0

### PR DESCRIPTION
**Pull request by xorye**
Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.5.0 (April 30, 2020)
+
+### Enhancements
+
+ * Code snippets for MicroProfile fault tolerance annotations. See [LS#307](https://github.com/redhat-developer/quarkus-ls/issues/307)
+ * Support for MicroProfile Context Propagation properties. See [LS#242](https://github.com/redhat-developer/quarkus-ls/issues/242)
+ * Create Container Image snippets, update Quarkus datasource snippets. See [#227](https://github.com/redhat-developer/vscode-quarkus/issues/227)
```
Comment (xorye) [inline CHANGELOG.md]: The PR for that issue is not merged yet

**Comment by xorye** [comment]
I don't think I should add this to the changelog: https://github.com/redhat-developer/vscode-quarkus/pull/245

**Comment by fbricon** [comment]
no it's irrelevant

File: CHANGELOG.md (hunk 1 above)
Comment (fbricon) [reply CHANGELOG.md]: It'll become "Update Quarkus datasource snippet" given https://github.com/redhat-developer/quarkus-ls/pull/310#discussion_r417490326

**Comment by xorye** [comment]
Changed the snippet enhancement to `Update Quarkus datasource snippet.`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.5.0 (April 30, 2020)
+
+### Enhancements
+
+ * Code snippets for MicroProfile fault tolerance annotations. See [LS#307](https://github.com/redhat-developer/quarkus-ls/issues/307)
+ * Support for MicroProfile Context Propagation properties. See [LS#242](https://github.com/redhat-developer/quarkus-ls/issues/242)
+ * Changed to Update Quarkus datasource snippet. See [#227](https://github.com/redhat-developer/vscode-quarkus/issues/227)
```
Comment (fbricon) [inline CHANGELOG.md]: Updated Quarkus datasource snippet

**Pull request merged**